node-operators/networks/run-a-node/systemd.md

```diff
@@ -334,6 +334,37 @@ For an overview of the flags used in the following start-up commands, plus addit
 
 ### Collator {: #collator }
 
+Beginning with v0.39.0, Moonbeam collator nodes no longer generate session keys automatically on start-up. Run the following command to generate and store on disk the session keys that will be referenced in the start-up command:
+
+=== "Moonbeam"
+
+    ```bash
+
+    /var/lib/moonbeam-data key generate-node-key --file /var/lib/moonbeam-data/node-key
```

Comment:
This is calling the data dir not the binary

Reply:
Fixed per our conversation. Thanks!